src/views/registration-js/attendees/fields/telephone.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/telephone.php
+ *
+ * @since   TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Telephone
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 2d26a0aac8a242384c007657010c6941</sub>